connectivity/libraries/ppp/source/ppp_service.cpp

```diff
@@ -60,7 +60,7 @@ extern "C" { // "pppos.h" is missing extern C
 /* Timeout to wait for PPP connection to be terminated
  * (LCP Terminate-Request is answered with Terminate-Ack)
  */
-#define PPP_TERMINATION_TIMEOUT           30000
+constexpr auto PPP_TERMINATION_TIMEOUT = 30000ms;
```

Comment:
Can we leave it as a macro or does it have to be turned to a constexpr?

Reply:
why keep a macro? constexpr can be used at compile time by the compiler to provide useful warnings.